string label        # 物体类别
float32 confidence # 置信度
float32 x         # 3D点 x 坐标
float32 y         # 3D点 y 坐标
float32 z         # 3D点 z 坐标